packages/spiderfier/index.ts

```diff
@@ -0,0 +1,7 @@
+// main modules
+export * from './directives';
+export * from './services';
+
+// we explicitly export the module here to prevent this Ionic 2 bug:
+// http://stevemichelotti.com/integrate-angular-2-google-maps-into-ionic-2/
```

Comment:
this bug is already solved (I know that this comment still in the core page :) ) we can remove this here.